api/google-search/src/main/java/com/google/search/googlesearch/services/SearchAutoCompleteService.java

```diff
@@ -0,0 +1,71 @@
+package com.google.search.googlesearch.services;
+
+import java.util.ArrayList;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class SearchAutoCompleteService {
+
+	private ArrayList<String> keywordArray;
```

Comment:
Follow the practise - code to interface not implementation

Reply:
Done